README.rst

```diff
@@ -30,13 +30,248 @@
 Triton Inference Server
 =======================
 
-    **NOTE: You are currently on the r20.09 branch which tracks
-    stabilization towards teh next release. This branch is not usable
-    during stabilization.**
-
 .. overview-begin-marker-do-not-remove
 
+riton Inference Server provides a cloud inferencing solution
```

Comment:
It seems like 'T' is missing.

Reply:
Good catch, @dzier  please fix before merging.